db/migrate/20190716110520_create_the_backend_apis_of_services.rb

```diff
@@ -0,0 +1,22 @@
+class CreateTheBackendApisOfServices < ActiveRecord::Migration
+  def change
+    say_with_time 'Migrating proxies api_backend to slugs...' do
+      reversible do |dir|
+        dir.up do
+          Service.transaction do
+            Service.find_each do |service|
+              service.find_or_create_first_backend_api!
```

Comment:
It is inside a loop so I will add just one of them, with Service ID 1 and Account ID 1 (its provider):
```sql
SELECT  * FROM accounts WHERE id = 1 LIMIT 1
SELECT  * FROM backend_api_configs WHERE service_id = 1  ORDER BY id ASC LIMIT 1
SELECT  * FROM proxies WHERE service_id = 1 LIMIT 1
BEGIN
  SELECT  1 AS one FROM backend_apis WHERE (system_name = 'master_service' AND account_id = 1) LIMIT 1
  INSERT INTO `backend_apis` (`system_name`, `name`, `description`, `private_endpoint`, `account_id`, `created_at`, `updated_at`) VALUES ('master_service', 'Master Service Backend API', 'Backend API of Master Service', 'https://echo-api.3scale.net:443', 1, 'YYYY-MM-DD HH:MM:SS', 'YYYY-MM-DD HH:MM:SS')
COMMIT
BEGIN
  INSERT INTO `backend_api_configs` (`backend_api_id`, `service_id`, `created_at`, `updated_at`) VALUES (1, 1, 'YYYY-MM-DD HH:MM:SS', 'YYYY-MM-DD HH:MM:SS')
COMMIT
```